tests/integration/test_verify.py

```diff
@@ -16,6 +17,24 @@ def test_verify_valid_sources(source_dir):
     assert exit_code == 0
 
 
+@pytest.mark.parametrize('source_dir,output', [
+    ("tests/test_files/yaml_source_dir/valid_yamls_with_multiple_crds",
+     "tests/bundle.yaml"),
+])
+def test_verify_with_output_file(source_dir, output):
+    process = subprocess.Popen(f'operator-courier verify {source_dir} --output={output}',
+                               shell=True,
+                               stdout=subprocess.PIPE,
+                               stderr=subprocess.STDOUT)
+    exit_code = process.wait()
+    assert exit_code == 0
+
+    fileCreated = os.path.isfile(output)
```

Comment:
I don't think it matters what the file is, we are creating it if it doesn't exist and overwriting it if it does. Whether they want to name their file with the correct file extension isn't really important, considering this is just a debug feature anyway.

Reply:
Yeah but I'm not talking about the file name/extension. I'm thinking as integration tests, we should test the functionality/feature of the application. For this specific test case, It would be ideal if we can check the content of the output matches what we would expect. We may not need to go that far, but currently even an empty file an pass the check.